src/galileo/handlers/langchain/async_handler.py

```diff
@@ -123,6 +123,7 @@ async def _log_node_tree(self, node: Node) -> None:
                 output=output,
                 model=node.span_params.get("model"),
                 temperature=node.span_params.get("temperature"),
+                tools=node.span_params.get("tools", None),
```

Comment:
nitpick: https://docs.python.org/3/library/stdtypes.html#dict.get
default=None, so we can just simplify to:
```suggestion
                tools=node.span_params.get("tools"),
```